frameworks/helloworld/src/main/dist/sidecar.yml

```diff
@@ -25,6 +25,7 @@ pods:
           # Note, this is very high so that the test test_sidecar.test_toxic_sidecar_doesnt_trigger_recovery
           # will always have an empty recovery plan.
           SLEEP_DURATION: 10000
+          CONFIG_SLEEP_DURATION: {{SLEEP_DURATION}}
```

Comment:
Wow, now I'm really confused. Does the `SLEEP_DURATION` on line 27 have anything to do with the `{{SLEEP_DURATION}}` on 28? Because it looks as if we allow setting it in service options, while actually don't, because we hardcode it to 10000 on line 27?